src/Model/Asset/MonitoredAssetOptions.php

```diff
@@ -138,12 +139,14 @@ public function toArray(): array
     public function toBytes(): string
     {
         return implode('', [
-            // TODO: Feeds Array
-            $this->getFeeds() ? '01' : '00',
+            $this->getFeeds() ?
+                VarInt::encodeDecToHex(sizeof($this->getFeeds()))
+                . '' // TODO array_map each element toBytes()
```

Comment:
unresolved todo

Reply:
Unknown structure of array, can't fix now.